aws/src/services/S3Storage.test.ts

```diff
@@ -125,7 +125,7 @@ describe("aws storage when call write should", () => {
       Bucket: input.container,
       Key: input.path,
       Body: readableBody,
-      ContentLength: readableBody.readableLength
+      ContentLength: readableBody.readableLength || (readableBody as any)._readableState.length
```

Comment:
Let's see what other solutions we can find.  The `readableState` is not exposed as a public member somewhere?